plugins/services/src/js/reducers/serviceForm/Containers.js

```diff
@@ -438,7 +443,8 @@ module.exports = {
   },
 
   FormReducer(state, {type, path = [], value}) {
-    const [base, index, field, secondIndex, name] = path; // eslint-disable-line no-unused-vars
+    // eslint-disable-next-line no-unused-vars
+    const [_, index, field, secondIndex, name] = path;
```

Comment:
in general I prefer having underscore prefix ignored in eslint so it would be
`const [_base, index, field, secondIndex, name] = path;` Shall we bring this to the next sync?